src/azure-cli/azure/cli/command_modules/synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -346,6 +347,57 @@ def test_sql_pool_audit_policy(self):
                      self.check('state', 'Enabled')
                  ])
 
+    @record_only()
+    def test_sql_aad_admin(self):
+        self.kwargs.update({
+            'location': 'eastus',
+            'workspace': 'testsynapseworkspace',
+            'rg': 'rg',
+            'user-name': 'fakeuser',
+            'object-id': '00000000-0000-0000-0000-000000000000',
+            'user-email': 'fakeuser@fakedomain.com'
+        })
+        # Test create cmdlet
+        self.cmd('az synapse sql ad-admin create --workspace-name {workspace} --resource-group {rg} '
+                 '--display-name {user-name} --object-id {object-id}',
+                 checks=[
+                     self.check('login', self.kwargs['user-name'])
+                 ])
+
+        # Test show cmdlet
+        self.cmd('az synapse sql ad-admin show --workspace-name {workspace} --resource-group {rg}',
+                 checks=[
+                     self.check('login', self.kwargs['user-name']),
+                     self.check('name', 'activeDirectory')
+                 ])
+
+        # Test update cmdlet
+        self.cmd('az synapse sql ad-admin update --workspace-name {workspace} --resource-group {rg} '
+                 '--display-name {user-email}',
+                 checks=[
+                     self.check('login', self.kwargs['user-email'])
+                 ])
+        # Test delete cmdlet
+        self.cmd('az synapse sql ad-admin delete --workspace-name {workspace} --resource-group {rg} -y')
+        self.cmd('az synapse sql ad-admin show --workspace-name {workspace} --resource-group {rg}', expect_failure=True)
+
+    @record_only()
+    def test_sql_audit_policy(self):
+        self.kwargs.update({
+            'location': 'eastus',
+            'workspace': 'testsynapseworkspace',
+            'rg': 'rg',
+            'storage-account': 'teststorageforsynapse'
+        })
+
+        self.cmd('az synapse sql audit-policy update --state Enabled --storage-account {storage-account} '
+                 '--workspace-name {workspace} --resource-group {rg}')
+
+        self.cmd('az synapse sql audit-policy show --workspace-name {workspace} --resource-group {rg}',
+                 checks=[
+                     self.check('state', 'Enabled')
+                 ])
```

Comment:
1. could we also check the update command?
2. could we get first, update and then get again to see the value change?
3. please add update for all parameters.

Reply:
> 1. could we also check the update command?
> 2. could we get first, update and then get again to see the value change?
> 3. please add update for all parameters.

Thanks, I have updated the test for `az synapse sql audit-policy` and `az synapse sql pool audit-policy`